ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -134,6 +140,26 @@ public static boolean isSystemColumn(final ColumnName columnName, final KsqlConf
     return isSystemColumn(columnName, getPseudoColumnVersionFromConfig(ksqlConfig));
   }
 
+  public static int getPseudoColumnVersionFromConfig(final KsqlConfig ksqlConfig) {
```

Comment:
Decided to make this public. WDYT? I don't see a huge issue in exposing it, especially given it's temporary, and I can clean up some instances of repeated logic from this.